src/main/res/values/strings.xml

```diff
@@ -23,14 +23,14 @@
     <string name="compiled_for_debugging">Compiled for debugging</string>
     <string name="confirmation">Confirmation</string>
     <string name="connecting">Connecting…</string>
-    <string name="contains_non_free_media">Contains non-free media</string>
+    <string name="contains_non_free_media">Contains non-libre media</string>
     <string name="could_not_download_FORMAT">Could not download %s</string>
     <string name="could_not_sync_FORMAT">Could not sync %s</string>
     <string name="could_not_validate_FORMAT">Could not validate %s</string>
-    <string name="credits">Credits</string>
+    <string name="credits">Contributors</string>
     <string name="dark">Dark</string>
-    <string name="delete">Delete</string>
-    <string name="delete_repository_DESC">Delete the repository?</string>
+    <string name="delete">Remove</string>
+    <string name="delete_repository_DESC">Stop connecting to this repository?</string>
```

Comment:
```suggestion
    <string name="delete_repository_DESC">Stop looking for apps from this repository?</string>
```

Reply:
Nah, it's still "disable". The correct term is remove/delete.

Reply:
I don't think the distinction is important to the user.